### PR TITLE
Update plugins.md (Fixing couple of 404)

### PR DESCRIPTION
**Pull request by Aldo-f**
Fixing couple of 404

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -3341,7 +3341,7 @@ Allows display of the map in full-screen mode.
 	</tr>
 	<tr>
 		<td>
-			<a href="http://brunob.github.com/leaflet.fullscreen">leaflet.fullscreen</a>
+			<a href="https://github.com/brunob/leaflet.fullscreen">leaflet.fullscreen</a>
```
Comment (johnd0e) [inline docs/plugins.md:3344]:
```suggestion
			<a href="https://brunob.github.io/leaflet.fullscreen/">leaflet.fullscreen</a>
```

File: docs/plugins.md [hunk 2]
```diff
@@ -3350,7 +3350,7 @@ Allows display of the map in full-screen mode.
 	</tr>
 	<tr>
 		<td>
-			<a href="http://elidupuis.github.com/leaflet.zoomfs">leaflet.zoomfs</a>
+			<a href="https://github.com/elidupuis/leaflet.zoomfs">leaflet.zoomfs</a>
```
Comment (johnd0e) [inline docs/plugins.md:3353]:
```suggestion
			<a href="https://elidupuis.github.io/leaflet.zoomfs/">leaflet.zoomfs</a>
```

**Review by johnd0e — CHANGES_REQUESTED** [review]

**Comment by Aldo-f** [comment]
Wouldn't it be better to just link to the GitHub?

I think, for keeping it safe/consistent, that would be the better option.

Or is the ultimate goal to link all resources to an individual github.io page (if it exists)?

**Comment by johnd0e** [comment]
@Aldo-f 
Please resolve conflicts

**Comment by Aldo-f** [comment]
It seems the 404's are already fixed on master.
No need to fix these conflicts.

**Pull request closed without merging**